addons/godot-xr-tools/player/player_body.gd

```diff
@@ -678,25 +678,26 @@ func _update_ground_information(delta: float):
 	ground_angle = rad_to_deg(ground_collision.get_angle(0, up_gravity))
 	ground_node = ground_collision.get_collider()
 
-	# Select the ground physics
-	var physics_node := ground_node.get_node_or_null("GroundPhysics") as XRToolsGroundPhysics
-	ground_physics = XRToolsGroundPhysics.get_physics(physics_node, default_physics)
+	if ground_node!=null:
```

Comment:
Same here, just change to `if ground_node:`

Reply:
(though if I am not mistaken, `XRToolsGroundPhysics.get_physics` already does a null check and returns default_physics if physics_node is not usable, so possibly this fix is incorrect.

Reply:
IDK about `XRToolsGroundPhysics.get_physics`, but when I was developing my project it was one of points where I got null exception so mayby it failed to do so ?